content/docs/getting-started/cms-set-up.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Set Up the CMS
+prev: /docs/getting-started/introduction
+next: /docs/getting-started/edit-content
+---
+
+In this step, we will install `tinacms`, create a CMS instance, wrap our app in the `TinaProvider`, and configure the CMS options.
+
+## Install _tinacms_
+
+Install the `tinacms` package. This is the core package that you will need anytime you're using Tina. You'll also need to install `styled-components` to render the editing UI properly.
+
+```bash
+yarn add tinacms styled-components
+```
+
+## Create a CMS instance, add _TinaProvider_
```

Comment:
It would be good to mention that at the end of this section the CMS will not be visible